apps/src/lab2/views/components/versionHistory/VersionHistoryButton.tsx

```diff
@@ -1,5 +1,5 @@
 import classNames from 'classnames';
-import React, {useState} from 'react';
+import React, {useCallback, useState} from 'react';
```

Comment:
The VersionHistory* mods are just adding useCallback and React.memo to cut down on the number of re-renders.